internal/impl/devbox.go

```diff
@@ -384,48 +386,33 @@ func (d *Devbox) GenerateEnvrc(force bool, source string) error {
 		)
 	}
 	// confirm .envrc doesn't exist and don't overwrite an existing .envrc
-	if cmdutil.Exists("direnv") {
-		// prompt for direnv allow
-		var result string
-		isInteractiveMode := isatty.IsTerminal(os.Stdin.Fd())
-		if isInteractiveMode {
-			prompt := &survey.Input{
-				Message: "Do you want to enable direnv integration for this devbox project? [y/N]",
-			}
-			err := survey.AskOne(prompt, &result)
-			if err != nil {
-				return errors.WithStack(err)
-			}
+	if source == "generate" {
```

Comment:
Not from this PR, but what is the purpose of this brancing logic? Don't we always want to do this check?

I'm a afraid if we don't do this check during `init` and then we add packages manually to devbox.json then these errors might not show up because we are calling it from inside a file and discarding the stderr.